Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
Because it is a logic that we need to use in different places. I.e. we need it everytime we need to draw an AABB, so we need it in both GameObject and Quadtree Logic. It's just a simple helper function. Why so dramatic?